bin/ch/$262.js

```diff
@@ -0,0 +1,24 @@
+R"(
```

Comment:
see https://github.com/Microsoft/ChakraCore/blob/master/build.sh#L536 

I had wrote a small python tool to convert string to `c char array` on pre build step. We can use the same approach here.